examples/scripts/enrichment/validate_addresses_from_file.py

```diff
@@ -0,0 +1,101 @@
+"""An example script to validate address data from disk and save results on disk."""
+import argparse
+from dataclasses import fields
+from typing import List
+
+import pandas as pd
+
+import tamr_toolbox as tbox
+from tamr_toolbox.enrichment.enrichment_utils import join_clean_tuple
+
+
+def main(
+    mapping_folder: str,
+    address_columns: List[str],
+    path_to_csv_to_validate: str,
+    path_to_validated_csv: str,
+) -> None:
+    """Validate data located on disk and save results to disk.
```

Comment:
This example is a good place to specify how the API key needs to be stored.

Reply:
Also important to specify since your example allows for a config file but your get_maps_client function assumes an environment variable. Worth converting to a config variable?

Reply:
Yes, perhaps that would be better. I can see about doing that.